src/components/Item.vue

```diff
@@ -27,7 +27,6 @@
 </template>
 
 <script>
-import { timeAgo } from '../util/filters'
```

Comment:
filter is also function, and it's used inside: https://github.com/vuejs/vue-hackernews-2.0/blob/master/src/components/Item.vue#L37

Reply:
but I think filter function has registered at: [https://github.com/vuejs/vue-hackernews-2.0/blob/master/src/app.js#L13](https://github.com/vuejs/vue-hackernews-2.0/blob/master/src/app.js#L13)

Reply:
It is used as a pure function in script rather than a filter in template, I've pointed out its usage reference just before.